sources/install/package_web.sh

```diff
@@ -922,7 +922,22 @@ function package_web() {
     install_soapui                  # SoapUI is an open-source web service testing application for SOAP and REST
     install_sqlmap                  # SQL injection scanner
     install_sslscan                 # SSL/TLS scanner
+    install_Postman                 # Postman - API platform for testing APIs
     end_time=$(date +%s)
     local elapsed_time=$((end_time - start_time))
     colorecho "Package web completed in $elapsed_time seconds."
 }
+
+function install_Postman() {
+    colorecho "Installing postman"
+    curl -L https://dl.pstmn.io/download/latest/linux_64 -o /tmp/postman.tar.gz
+    tar -xf /tmp/postman.tar.gz --directory /tmp
+    rm /tmp/postman.tar.gz
+    chown -R root:root /tmp/Postman
+    mv /tmp/Postman /opt/tools/Postman
+    ln -s "/opt/tools/Postman/app/Postman" "/opt/tools/bin/postman"
+    apt install libsecret-1-0 -y
+    add-history postman
+    add-test-command "which postman"
```

Comment:
If possible, the test command should check that the tool is working properly by running it.

Reply:
I use "which postman" to avoid the test script to open the app (which force the user to close it in order to allow the test to continue), pretty much like Burp would. Burp using "which burpsuite" command, i decided to do the same.

Reply:
Yes, but burp's test command changes one day to make sure that the tool is working properly. A better solution should be found here I think

Reply:
For now, we can let this test command